cmd/nodepool/kubevirt/create.go

```diff
@@ -71,6 +74,10 @@ func (o *KubevirtPlatformCreateOptions) UpdateNodePool(_ context.Context, nodePo
 		return fmt.Errorf(`wrong value for the --network-multiqueue parameter. May be only "Enable" or "Disable"`)
 	}
 
+	if o.QoSClass != "" && o.QoSClass != hyperv1.QoSClassGuaranteed {
+		return fmt.Errorf(`wrong value for the --qos-class parameter. May be only "Guaranteed"`)
```

Comment:
make sure to change this to burstable and guaranteed if you add burstable as an option